icons/src/fullcolor/emblems/emblem-symbolic-link.svg

```diff
@@ -523,7 +523,7 @@
          height="15"
          width="15"
          id="rect4414"
-         style="display:inline;fill:#f9f9f9;fill-opacity:1;stroke:none;stroke-width:1;stroke-opacity:0.25098039;enable-background:new"
+         style="display:inline;fill:#1a1a1a;fill-opacity:1;stroke:none;stroke-width:1;stroke-opacity:0.25098039;enable-background:new"
```

Comment:
This is supposed to be #3D3D3D, not #1A1A1A

Reply:
I indicated the reason for choosing a darker color in https://github.com/ubuntu/yaru/pull/1151#issuecomment-460772929

Reply:
So you made a poll, your idea lost, and you ignored it?

Reply:
Moreover you used the color that was rejected by anyone, nice move :+1:

Reply:
Sorry, I didn't understand you.

Reply:
@Feichtmeier agreed with me that window background color in dark theme is also `#3d3d3d`. If I change emblem color to this color, then the circle become less noticeable in dark theme.